src/Helpers_TEST.cc

```diff
@@ -25,21 +25,27 @@ using namespace fuel_tools;
 /////////////////////////////////////////////////
 TEST(HelpersTEST, UriToPathNoAuthority)
 {
+#ifdef WIN32
```

Comment:
Please add a reference to the `uriToPath` issue here before merging.
```suggestion
// TO-DO: Update this test after ign-fuel-tools#204 is addressed
#ifdef WIN32
```